examples/lqr/lqr_experiment.py

```diff
@@ -15,11 +15,12 @@
 from safe_control_gym.utils.registration import make
 
 
-def run(gui=True, n_episodes=1, n_steps=None, save_data=False):
+def run(gui=False, plot=True, n_episodes=2, n_steps=None, save_data=False):
```

Comment:
Why is the default for `n_episodes` changed? Seems to be done for multiple controllers.

Reply:
Yeah this is something I was wondering. Ideally all the `_experiment.py` files would share a relatively similar interface and defaults. I added plot because I find it useful to have `gui` and `plot` separated, and I added `n_episodes=2` because I find it useful to test that we can still run multiple episodes and properly get the results and plots, as well as show off how easy it is to run multiple episodes. However, this is really a matter of preference. Happy to revert if you prefer to have another interface and defaults.

Reply:
I like the separation for `gui` and `plot` but i would keep `n_episodes=1`

Reply:
reverted to `n_episodes=1`